### DISCUSSION TITLE
Kubernetes Helm Support

### DISCUSSION THREAD

**Discussion by cabrinha** [zed-industries/zed#7260] [Feature Requests]
Hello, I'd love to get Helm support added into Zed, but it's a little tricky due to the fact that Helm is a combination of Go templating and YAML. Tree-sitter issue: https://github.com/ikatyang/tree-sitter-yaml/issues/36

I've opened a PR, but have no idea how to go about this: https://github.com/zed-industries/zed/pull/7259

There is no tree-sitter for Helm specifically, but go-templating and yaml tree-sitters do exist.

**Comment by cabrinha** [comment]
There is a [helm-ls language server](https://github.com/mrjosh/helm-ls?tab=readme-ov-file#yaml-language-server-config), but there is no tree-sitter for it. I'm just not sure how get that language server working in Zed.

@tml @maxdeviant if you could provide any insight on how I could get started, I'd appreciate it.

**Comment by cabrinha** [comment]
Anyone? 😆

**Reply by SomeoneToIgnore** [reply]
Anyone what?
https://github.com/zed-industries/zed/pulls?q=is%3Apr+is%3Aopen+helm

**Reply by cabrinha** [reply]
Anyone able to tell me how to combine YAML and go-template language servers / tree-sitters to enable Helm support in Zed? 😆

**Reply by SomeoneToIgnore** [reply]
Oh yeah, writing a comment in the PR, but in short: not me, sorry, have no idea yet.

**Comment by romans-ovo** [comment]
I'd really love to help as I'm also interested in Helm syntax, but my knowledge is non-existant. 

Did you try go-template?

https://github.com/nvim-treesitter/nvim-treesitter/discussions/1917

**Reply by cabrinha** [reply]
I'm not sure how to approach this in Rust/Zed

**Comment by cabrinha** [comment]
We've got the issue transferred over to Extensions: https://github.com/zed-industries/extensions/issues/186

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
